src/Card/Service.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace PodPoint\Payments\Card;
+
+use PodPoint\Payments\Card;
+use PodPoint\Payments\Customer;
+use PodPoint\Payments\Token;
+
+interface Service
+{
+    /**
+     * Attach a card to a customer.
+     *
+     * @param Token $cardToken
+     * @param Token $customer
+     *
+     * @return Card
+     */
+    public function attach(Token $cardToken, Token $customerToken): Card;
+
+    /**
+     * Remove a card.
+     *
+     * @param Token $cardToken
+     * @param Token|null $customerToken
+     */
+    public function remove(Token $token, $customerToken = null): void;
+
+    /**
+     * Create a card.
+     *
+     * @param Token $token
+     * @param array $params
+     *
+     * @return Card
+     */
+    public function create(Token $token, array $params = []): Card;
```

Comment:
what are the `params` ? would be good to avoid an array

Reply:
So this is the solution I found so that we aren't locking the possibilities to our use case (some might want to create cards that would require off session authentication, or other types of payment methods).  But if this package isn't about to go public I can remove it.